src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -0,0 +1,432 @@
+---
+title: Monitor your Flutter app
+tags:
+    - Flutter
+    - Mobile monitoring
+    - New Relic Mobile Flutter
+    - Get started
+metaDescription: "New Relic's Flutter agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+---
+
+import mobileFlutterOverview from 'images/mobile_screenshot-full_flutter-overview.png'
+
+Monitor your Flutter application with our New Relic Flutter agent. Capture Dart errors, track specific network requests, and use distributed tracing to drill down into handled exceptions, and create custom tracking events and metrics to fully capture your end users' experience.
+
+<img
+  title="mobile UI for flutter "
+  alt="mobile UI for flutter "
+  src={mobileFlutterOverview}
+/>
+
+<figcaption>
+  
+</figcaption>
+
+Ready to get started? Follow the steps below! 
+
+
+## Step 1: Review the requirements
+
+Before you start the install process, make sure your Flutter app meets these version requirements:
+
+* Flutter 2.5 and higher.
+* Dart 2.16.2 and higher, up to 2.9.9
+
+For Android-native apps:
+    * Android API 24+
+    * See [Android native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
+
+For iOS-native apps:
+    * See [IOS native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
+
+## Step 2: Install the Flutter agent 
+
+First, you'll need to install the New Relic plugin into your dart project. In your `pubspec.yaml`, add the following to `dependencies`: 
+
+```dart
+dependencies:
+    newrelic_mobile: 0.0.1-dev.10
+```
+
+## Step 3: Copy your application token #app-token
+
+The application token is used for New Relic to authenticate your Flutter app's data. 
+
+To view and copy your application token in the New Relic UI:
+
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Add data**, then click **Mobile**.
+2. Select your Flutter app. 
+3. Go to **Settings > Application**, and copy the displayed **Application token**.
```

Comment:
@ndesai-newrelic Is it assumed they've already created an app in NR for their flutter app?